configure.ac

```diff
@@ -495,8 +495,10 @@ AC_ARG_ENABLE(versioning,
 
 if test "$enable_versioning" = "yes" ; then
    ABIVERSIONFLAGS="-version-info \$(ABIVERSION)"
+   ABI_ABIVERSIONFLAGS="-version-info libmpi_abi_so_verion_m4"
 else
    ABIVERSIONFLAGS="-avoid-version"
+   ABI_ABIVERSIONFLAGS="-avoid-version"
```

Comment:
@hzhou @raffenet Would this mean that a build from release tarball vs a build from a git close will produce different version numbers? Are you absolutely sure you want this behavior?